playground/frontend/lib/constants/params.dart

```diff
@@ -21,5 +21,6 @@ const kIsEditableParam = 'editable';
 const kSourceCode = 'code';
 const kContextLine = 'line';
 const kIsEmbedded = 'embedded';
+const kSnippetId = 'snippetId';
```

Comment:
Please rename it to `kSnippetIdParam` to align with the first two. Also we should keep the constants with correct names sorted alphabetically. Currently `kExampleParam` and `kIsEditableParam` are named correctly. This one should go after them. We will reorder other constants as we go about renaming them.

Reply:
Fixed.